bath-se2-group3/NewBank
# Addition | Add a New Account

# [Addition | Add a New Account](https://trello.com/c/wk386Ab5/19-addition-protocol-newaccount-create-new-account-for-existing-customer)

## What Does the Code Do
This PR allows a user to add a new account, for example a Savings account. To do this, a user should type `CREATEACCOUNT <Account_Name> <Starting_Balance>

## How to Test
- Login to NewBank
- Type `HELP` to see the help information
- Type `CREATEACCOUNT <Account_Name> <Starting_Balance>
  - For Example: `CREATEACCOUNT RMSavings 100`
- If successful, the Output should look like this:
<img width="491" alt="Screenshot 2022-07-19 041440" src="https://user-images.githubusercontent.com/52407765/179656716-ed271f06-b26d-4b08-bff7-b9765b1e4eb7.png">

## Future Development
- Possible additions I thought about:
  - Allow doubles input, i.e. currently a user can't enter 11.83
  - Format Money with Currency / Two Decimal Places
  - Add optional input for the starting balance (i.e. only accepting one argument)
  - Format `accountsToString()` method with new lines.

## Relevant Resources
- N/A
